### PR TITLE
Updates docker and some vscode settings

### PR DESCRIPTION
**Pull request by tomatau**
@believer-ufa

### CODE REVIEW

**Comment by believer-ufa** [comment]
why you lowered versions on free dependencies?

**Comment by tomatau** [comment]
which? it was a mistake -- my local branch was getting old so I rushed it out because you were making changes today :D

**Comment by tomatau** [comment]
https://github.com/tomatau/breko-hub/pull/297

File: package.json [hunk 1]
```diff
@@ -61,28 +61,22 @@
...
     "babel-plugin-provide-modules": "^1.0.3",
     "babel-plugin-ramda": "^1.6.1",
-    "chokidar": "^2.0.3",
     "classnames": "^2.2.6",
-    "clean-webpack-plugin": "^0.1.19",
-    "css-loader": "^1.0.0",
     "css-modules-require-hook": "^4.2.3",
-    "cssnano": "^4.0.0",
-    "debug": "^4.0.0",
+    "debug": "^3.1.0",
```
Comment (believer-ufa) [inline package.json]: from 4.x to 3.x

File: package.json [hunk 2]
```diff
@@ -116,33 +106,26 @@
...
     "redux-devtools-extension": "^2.13.2",
     "redux-logger": "^3.0.6",
     "redux-promise-middleware": "^2.5.0",
     "redux-saga": "^0.16.0",
     "redux-thunk": "^2.3.0",
     "redux-via-socket.io": "~0.3.0",
-    "reselect": "4.0.0",
-    "rimraf": "^2.6.2",
-    "sass-loader": "^7.0.3",
+    "reselect": "3.0.1",
```
Comment (believer-ufa) [inline package.json]: from 4.x to 3.x

File: package.json [hunk 3]
```diff
@@ -151,32 +134,49 @@
...
+    "enzyme": "^3.5.1",
+    "enzyme-adapter-react-16": "^1.4.0",
     "enzyme-to-json": "^3.3.4",
     "eslint": "^5.0.0",
     "eslint-plugin-import": "^2.12.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-mocha": "^5.0.0",
     "eslint-plugin-react": "^7.9.1",
-    "fetch-mock": "^7.0.2",
+    "fetch-mock": "^6.4.4",
```
Comment (believer-ufa) [inline package.json]: from 7.x to 6.x

Comment (tomatau) [reply package.json]: these are both back up to latest in master now

**Pull request merged**